src/main/java/com/example/onlinebookstore/repository/BookRepository.java

```diff
@@ -1,13 +1,9 @@
 package com.example.onlinebookstore.repository;
 
 import com.example.onlinebookstore.model.Book;
-import java.util.List;
 import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
 
-public interface BookRepository {
-    Book save(Book book);
-
-    List<Book> findAll();
-
-    Optional<Book> findById(Long id);
+public interface BookRepository extends JpaRepository<Book, Long> {
+    Optional<Book> getBookById(Long id);
```

Comment:
```suggestion
    Optional<Book> findById(Long id);
```

Reply:
after these changes I have a warning: 
![image](https://github.com/SalnikovMarkiian/OnlineBookStore/assets/98831511/d301f568-4a4d-4042-bacc-17ea8ed5fbbc)

Reply:
This warning tells you that findById should not return null value. But as your method return Optional you don't have to worry about it